### DISCUSSION TITLE
OR-Tools usage for commercial purposes

### DISCUSSION THREAD

**Discussion by raphaelboudreault** [google/or-tools#3215] [Build and configuration issues] [answered]
Just to make sure I understand well. Does installing OR-Tools from the current binaries for a **commercial use**, but only use the CP-SAT solver to solve our models, allowed by the licensing requirements? Or the only way would be to do a custom build of OR-Tools from source with the `USE_SCIP=OFF` option? If the latest is true, it seems to be that the academic license warning should also appear in the _Installing OR-Tools for X from Binary on Y_ sections of the documentation.

**Comment by lperron** [comment, marked as answer]
answered in a different thread.

To use SCIP, you need a license.
If you are not using it, you do not need a license.
